Providers/Scripts/2.4x-2.5x/Scripts/Tests/test_nxOMSAutomationWorker.py

```diff
@@ -228,28 +228,29 @@ def test_get_stray_worker_and_manager_pids(self):
         stray_processes = nxOMSAutomationWorker.get_stray_worker_and_manager_wsids(processes, self.workspace_id)
         self.assertFalse(stray_processes)
 
-    def test_is_any_1_3_process_running(self):
```

Comment:
What was intended to be checked in as 1.4  was held off for 1.5 and 1.4 became to hotfix to 1.3.
The resource already handles a case where old versions are running, step from 1.4 to 1.5 has to be treated specially because the resource schema changed, and a worker manager was introduced. This special function is needed to detect version 1.4 because the first run of 1.5 will have to apply special logic to deal with it.